build.gradle.kts

```diff
@@ -34,21 +35,28 @@ java {
   targetCompatibility = JavaVersion.VERSION_11
 }
 
+val compileKotlin: KotlinCompile by tasks
```

Comment:
You can also write something like:
`tasks.withType<KotlinCompile>().configureEach {
    kotlinOptions.jvmTarget = "1.8"
}`

Reply:
It in fact looks better.